Heavy-Division/B78XH
# feat: New Push Button Font

# New Push Button Textures

# Description 
I have replaced the old fonts on pushbuttons with new ones, drastically improved ones which look far more realistic than their original counterpart.

# Source material



# Testing Instructions 
Load in sim as normal, observe pushbutton textures and give feedback on them.

Please do keep in mind, some pushbutton textures may be the old font, this is due to their text not being in the decal set and I cannot find them whatsoever. Also do keep in mind that 1 of the decals on the OVHD towards the back may have broke, the switch itself is INOP and this was one of the sacrifices I had to do, I am severely limited to what I can and cannot do due to the model being locked.

# Test Results
![image](https://user-images.githubusercontent.com/70115175/214808102-4279122a-19e6-499b-9b4a-9be33fa4425e.png)

# Discord Username
KO#2134
